app/products/calls/screens/call_other_actions/call_other_actions.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 import React, {useCallback} from 'react';
-import {injectIntl, intlShape} from 'react-intl';
+import {injectIntl} from 'react-intl';
```

Comment:
if no internalization is used, why do we need to still inject Intl ?

Reply:
Originally I've tried to remove it completely and got stuck on a tsc error:

```
error TS2590: Expression produces a union type that is too complex to represent.
```

on an apparently unrelated file. Then I just thought of leaving it be for now since the plan is to use localization at some point anyway.